doi-proxy/src/test/java/no/unit/nva/doi/CrossRefClientTest.java

```diff
@@ -98,4 +108,24 @@ private void targetURlReturnsAValidUrlForDoiStrings(String doiPrefix)
         String output = crossRefClient.createUrlToCrossRef(doiURL).toString();
         assertThat(output, is(equalTo(expected)));
     }
+
+    private HttpClient mockHttpClientWithNonEmptyResponse() throws IOException {
```

Comment:
moved from the abstract class `AbstractLambdaTest`since they are used only here.  The class `AbstractLambdaTest` has been deleted, as its name is incorrect and its existence has no meaning.